Makefile

```diff
@@ -20,7 +20,10 @@ lint:
 
 test-build:
 	$(call eachmod,go build -o /dev/null ./...)
-	docker build -t ipfs/testground .
 
-test:
-	$(call eachmod,go test -v $(GOTFLAGS) ./...)
+test-quick:
```

Comment:
We're better off using [`testing.Short()`](https://golang.org/pkg/testing/#Short) than introducing this ad-hoc config.